src/components/overview/Overview.vue

```diff
@@ -31,44 +31,44 @@
 		</div>
 
 		<div v-else-if="isValidFilter" class="overview">
-			<div v-if="cardsByDueDate.overdue.length > 0" class="dashboard-column">
+			<div v-if="this.assignedCardsDashboard.length > 0" class="dashboard-column">
 				<h3>{{ t('deck', 'Overdue') }}</h3>
-				<div v-for="card in cardsByDueDate.overdue" :key="card.id">
+				<div v-for="card in this.assignedCardsDashboard.overdue" :key="card.id">
 					<CardItem :id="card.id" />
 				</div>
 			</div>
 
 			<div class="dashboard-column">
 				<h3>{{ t('deck', 'Today') }}</h3>
-				<div v-for="card in cardsByDueDate.today" :key="card.id">
+				<div v-for="card in this.assignedCardsDashboard.today" :key="card.id">
 					<CardItem :id="card.id" />
 				</div>
 			</div>
 
 			<div class="dashboard-column">
 				<h3>{{ t('deck', 'Tomorrow') }}</h3>
-				<div v-for="card in cardsByDueDate.tomorrow" :key="card.id">
+				<div v-for="card in this.assignedCardsDashboard.tomorrow" :key="card.id">
```

Comment:
```suggestion
				<div v-for="card in assignedCardsDashboard.tomorrow" :key="card.id">
```

Node js warnings seem valid for all the occurrences ;)